### PR TITLE
Homework1 task1 add

### PR DESCRIPTION
**Pull request by IuliiaLanchuk**
Homework1 task1 add

### CODE REVIEW

File: homework1/task01_sample_project/calculator/__init__.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-This is confest.py docstring.
+This is __init__.py docstring.
 
 Some text to pass flake8.
 """
```
Comment (Cjkjvfnby) [inline homework1/task01_sample_project/calculator/__init__.py]: Just remove `flake8-docstrings` `flake8-annotations` from requirements, pre-commit config, and uninstall them via pip.

File: homework1/task01_sample_project/calculator/calc.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+def check_power_of_2(a: int) -> bool:
+    """
+    I'm check is value a power of two.
+
+    :param a: input value
+    :return: bool
+    """
+    return type(a) == int and a != 0 and not (bool(a & (a - 1)))
```
Comment (Cjkjvfnby) [inline homework1/task01_sample_project/calculator/calc.py]: Please use `isinstance`

File: homework1/task01_sample_project/calculator/calc.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+5. then copy the `sample_project` into the directory.
+6. fix all bugs in the `sample_project` code
+7. write an extra test for each found bug
+**Note**: as we said, any hw, which does not pass `isort --profile black --check` and `black --check`, will be rejected
+"""
+
+
+def check_power_of_2(a: int) -> bool:
+    """
+    I'm check is value a power of two.
```
Comment (Cjkjvfnby) [inline homework1/task01_sample_project/calculator/calc.py]: I'd recommend reading that recommendations https://www.python.org/dev/peps/pep-0257/

File: homework1/task01_sample_project/tests/test_calculator.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        (12, False),
+        (8, True),
+        (3, False),
+        (0, False),
+        (4, True),
+        (1 / 8, False),
+    ],
+)
+def test_power_of_2(value: int, expected_result: bool):
+    actual_result = check_power_of_2(value)
```
Comment (Cjkjvfnby) [inline homework1/task01_sample_project/tests/test_calculator.py]: You can inline this variable.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: homework1/task01_sample_project/calculator/__init__.py (hunk 1 above)
Comment (IuliiaLanchuk) [reply homework1/task01_sample_project/calculator/__init__.py]: removed -> uninstalled

File: homework1/task01_sample_project/calculator/calc.py (hunk 2 above)
Comment (IuliiaLanchuk) [reply homework1/task01_sample_project/calculator/calc.py]: changed

File: homework1/task01_sample_project/calculator/calc.py (hunk 3 above)
Comment (IuliiaLanchuk) [reply homework1/task01_sample_project/calculator/calc.py]: thanks, done

File: homework1/task01_sample_project/tests/test_calculator.py (hunk 4 above)
Comment (IuliiaLanchuk) [reply homework1/task01_sample_project/tests/test_calculator.py]: inlined

File: homework1/task01_sample_project/calculator/__init__.py (hunk 1 above)
Comment (Cjkjvfnby) [reply homework1/task01_sample_project/calculator/__init__.py]: Please remove this docstring.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]
All OK, please fix the last comment and re-request reveiw

File: homework1/task01_sample_project/calculator/__init__.py (hunk 1 above)
Comment (IuliiaLanchuk) [reply homework1/task01_sample_project/calculator/__init__.py]: removed

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**